src/sass/_page_header.scss

```diff
@@ -24,7 +24,6 @@
 
   .page-header__base-actions {
     align-self: flex-start;
-    flex: 1;
```

Comment:
If we remove the flex, we can also remove the align self and the whole block, as it won't have any effect.